assets/components/atoms/link/link.scss

```diff
@@ -12,7 +12,7 @@ a {
   }
 
   &:active {
-    color: $wild-watermelon;
+    color: $epflred;
```

Comment:
So let's refactor it this way since `$epflred` is meaningless in this context and we have a variable for links color:

```scss
a {
  text-decoration-color: $link-hover-color;
  transition:
    text-decoration-color 0.2s ease-in-out,
    color 0.2s ease-in-out;

  @include hover-focus-active {
    color: $link-hover-color;
    text-decoration-color: $link-color;
  }

  &.text-muted:hover {
    color: $link-hover-color !important;
  }
}
```